src/app/register/components/form-visibility/form-visibility.component.html

```diff
@@ -27,21 +27,30 @@ <h3 class="mat-subheading-2" i18n="@@register.visibility">
     >
       <mat-radio-button
         [value]="visibilityOptions[0]"
-        i18n="@@register.VisibilityEveryone"
+        class="orcid-radio-button-with-icons"
       >
-        Everyone (87% of users choose this)
+        <img src="./assets/vectors/visibility-everyone.svg" />
+        <ng-container i18n="@@register.VisibilityEveryone"
+          >Everyone (87% of users choose this)</ng-container
+        >
       </mat-radio-button>
       <mat-radio-button
         [value]="visibilityOptions[1]"
-        i18n="@@register.VisibilityTrusted"
+        class="orcid-radio-button-with-icons"
       >
-        Trusted parties (5% of users choose this)
+        <img src="./assets/vectors/visibility-trusted-source.svg" />
```

Comment:
Attribute alt it is missing for accessibility in all the images

Reply:
So instead of adding alt labels, and since these icons are purely decorative icons I am setting those as hidden for accessible tools. 
In this way those do no get read by screen readers and create a confusing experience.